radix-engine-interface/src/address/hrpset.rs

```diff
@@ -13,8 +13,9 @@ pub struct HrpSet {
 
     normal_component: String,
     account_component: String,
+    identity_component: String,
 
-    system: String,
+    system_component: String,
```

Comment:
Should we just remove this?

Reply:
Yup good point!

Reply:
Actually, we are still using this for `EpochManager` and `Clock` and `Validator`. I will delay updating this until we get agreement with product side what's the right hrp to use for these.